src/PartialCache.php

```diff
@@ -55,7 +55,7 @@ public function __construct(View $view, Cache $cache, CacheManager $cacheManager
 
         $this->cacheKey = $config->get('partialcache.key');
         $this->cacheIsTaggable = is_a($this->cacheManager->driver()->getStore(), TaggableStore::class);
-        $this->enabled = $config->get('partialcache.enabled');
+        $this->enabled = filter_var($config->get('partialcache.enabled'),FILTER_VALIDATE_BOOLEAN);
```

Comment:
Could `(bool)` work here?

Reply:
No, because PHP translates string "false" as true. 
>>> (bool)"true"
=> true
>>> (bool)"false"
=> true

Btw, `filter_var` function takes care of this, and allows you pass boolean as string, and correctly recognizes it.